src/api/spotifyApi.js

```diff
@@ -1,33 +1,56 @@
-const client_id = '8a649d0e1cf74b1c89b6874845b17646';
-const client_secret = '401dcd9823934db3bae3337efb7320a5';
+const client_id = "8a649d0e1cf74b1c89b6874845b17646";
```

Comment:
General best practice is to have these outside the repo. Secrets should always be stored in environment variables as opposed to hard coded